app/views/relationships/index.html.haml

```diff
@@ -0,0 +1,21 @@
+%section.people
+  %header
+    %h2 People I Follow
+  %table.table
+    %thead
+      %tr
+        %th(width="30%")
+        %th(width="20%") Videos in Queue
+        %th(width="20%") Followers
+        %th(width="30%") Unfollow
+    %tbody
+      - @relationships.each do |leader|
+        %tr
+          %td
+            %img(src="http://www.gravatar.com/avatar/#{Digest::MD5.hexdigest('#{leader.email}'.downcase)}?s=40")
```

Comment:
I would refactor this in 2 steps:

1. use `= image_tag ...`
2. create a helper method for that long gravatar url, its not good for the eyes to read such code. `gravatar_for(leader)` and do the magic in the helper method in ApplicationHelper for instance.